app/(overview)/page.tsx

```diff
@@ -1,7 +1,31 @@
+import CustomTable from "@/components/CustomTable/CustomTable";
+const DataTable = [
+  {
+    key: "1",
+    userName: "Mai Hồng Tín",
+    email: "tinhongmai@gmail.com",
+    creationDate: "2023-09-05",
+    event: "Event A",
+    team: "Team X",
+    role: "Leader",
+    flagSetting: true,
+  },
```

Comment:
Variable except the component name should be in camelCase. In addition, the property indicating whether to show the action buttons should be a property of the component, not of each record like this.